src/otx/algo/detection/mmconfigs/yolox_l.yaml

```diff
@@ -20,7 +20,7 @@ data_preprocessor:
     - 1.0
     - 1.0
     - 1.0
-  bgr_to_rgb: false
+  bgr_to_rgb: true
```

Comment:
Why do you change this? AFAIK, the input should be RGB currently for all tasks.

Reply:
In 1.x yolox-x, yolox-l, and yolox-s models' to_rgb value is False. And it seems 2.0 should be reverse of that. So it should be True. And also when I get trained weight from 1.x, bgr_to_rgb should be True to get proper mAP

Reply:
So, the input should be BGR, right?

Reply:
Yes, I guess the pretrained weights require BGR

Reply:
If so, please update the data pipelines of YOLO models rather than this change. It does not transparently reflect the true state of the model.
https://github.com/openvinotoolkit/training_extensions/blob/c1cec1a336e05b82260181d955a2ce4e89be6570/src/otx/config/data/default.yaml#L6

Furthermore, to become better, you might add the image color format checker (raise an error if the input is not BGR) here:
https://github.com/openvinotoolkit/training_extensions/blob/c1cec1a336e05b82260181d955a2ce4e89be6570/src/otx/algo/detection/yolox.py#L12-L18

Reply:
So you mean keep bgr_to_rgb False and change src/otx/recipe/detection/yolox_l.yaml's img_color_channel?

Reply:
Sure

Reply:
Done [1de25e3](https://github.com/openvinotoolkit/training_extensions/pull/2811/commits/1de25e37cb2f77690db2a15db27c21f1f922d04f) I'll look at color format checker after this PR